src/distilabel/pipeline/base.py

```diff
@@ -398,7 +398,9 @@ def dry_run(
         self._dry_run = False
         return distiset
 
-    def get_runtime_parameters_info(self) -> Dict[str, List[Dict[str, Any]]]:
+    def get_runtime_parameters_info(
+        self,
+    ) -> Dict[str, Union[List[Dict[str, Any]], Dict[str, Dict[str, Any]]]]:
```

Comment:
Wouldn't be more readable to have some alias for this kind of type hint? just a personal opinion

Reply:
yes, I'll add it.